app/models/topic.rb

```diff
@@ -52,6 +52,7 @@ def filtered_topic_thumbnails(extra_sizes: [])
 
   def thumbnail_info(enqueue_if_missing: false, extra_sizes: [])
     return nil unless original = image_upload
+    return nil unless original.filesize < SiteSetting.max_image_size_kb
```

Comment:
should be SiteSetting.max_image_size_kb * 1024 or smth like that - original.filesize is in bytes

Reply:
ooo yea, I'll check it out and update it. Thanks!